pywemo/ssdp.py

```diff
@@ -191,6 +187,43 @@ def __repr__(self):
             self.values.get('st', ''), self.values.get('location', ''))
 
 
+def build_ssdp_request(st, ssdp_mx):
+    ssdp_st = st or ST
+    return "\r\n".join([
+        'M-SEARCH * HTTP/1.1',
+        'ST: {}'.format(ssdp_st),
+        'MX: {:d}'.format(ssdp_mx),
+        'MAN: "ssdp:discover"',
+        'HOST: 239.255.255.250:1900',
+        '', '']).encode('ascii')
+
+
+def entry_in_entries(entry, entries):
+    if entry.description is not None:
+        device = entry.description.get('device', {})
+        mac = device.get('macAddress')
+        serial = device.get('serialNumber')
+    else:
+        mac = None
+        serial = None
+    # If we don't have a mac or serial, let's just compare objects instead:
+    if mac == None and serial == None:
```

Comment:
I would change these to is None instead of == None. In this case, either should work, but is None is generally considered the best practice and is more efficient than the == None check.

Reply:
You're right. Normally I run these sorts of things through a pep-8 check.